CLAUDE.md

```
@@ -0,0 +1,100 @@
+
+### Caching Strategy
+- Character/corporation/alliance data: 24-hour TTL
+- System information: 1-hour TTL
+- Notification deduplication: 30-minute window
+- Use `WandererNotifier.Cache` module for all cache operations
+
+### Feature Flags
+Features can be toggled via environment variables ending in `_ENABLED`:
+- `KILLMAIL_NOTIFICATION_ENABLED`
+- `SYSTEM_NOTIFICATION_ENABLED`
+- `CHARACTER_NOTIFICATION_ENABLED`
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)